lib/models/workouts/session.dart

```diff
@@ -42,10 +42,10 @@ class WorkoutSession {
   late String notes;
 
   @JsonKey(required: true, name: 'time_start', toJson: timeToString, fromJson: stringToTime)
-  late TimeOfDay timeStart;
+  late TimeOfDay timeStart = TimeOfDay.now();
```

Comment:
do we still need this?

Reply:
Since those are default values that are not constant, better move them to a new constructor, something like this, then we're good to go

```dart
WorkoutSession.now(){
    timeStart = TimeOfDay.now();
    timeEnd = TimeOfDay.now();
  }
```

Reply:
@rolandgeider, where would you like to call it so these fields will be initialized?

Just replace this line from gym_mode.dart:

`final _session = WorkoutSession();`
?

with this:

`final _session = WorkoutSession.now();`

Reply:
Yeah, exactly

Reply:
@rolandgeider, done.